server/views/pages/prison-register/prisonDetails.njk

```diff
@@ -65,13 +65,16 @@
   {% endif %}
 {% endmacro %}
 
-{% macro renderAddress(line1, line2, town, county, postcode, country) %}
+{% macro renderAddress(line1, line2, town, county, postcode, country, addressId, showAddWelshAddressLink) %}
   {{ line(line1) }}
   {{ line(line2) }}
   {{ line(town) }}
   {{ line(county) }}
   {{ line(postcode) }}
   {{ line(country) }}
+  {% if(isWelshPrison and showAddWelshAddressLink)%}
+    <a href="/prison-register/add-welsh-prison-address-start?prisonId={{prisonDetails.id}}&addressId={{addressId}}" class="govuk-link" data-qa="add-welsh-address-link-{{addressId}}">Add Welsh address</a>
+  {% endif %}
 {% endmacro %}
```

Comment:
only show the 'Add Welsh address' link if the prison is in Wales and the English address doesn't already have a Welsh address attached to it.